caching-service/src/main/java/org/zowe/apiml/caching/service/vsam/config/VsamConfig.java

```diff
@@ -37,6 +37,8 @@
     private int recordLength;
     @Value("${caching.storage.vsam.encoding:" + ZFileConstants.DEFAULT_EBCDIC_CODE_PAGE + "}")
     private String encoding;
+    @Value("${caching.storage.inmemory.size:100}")
```

Comment:
We should move this to the GeneralConfig and change the property name to caching.storage.size